packages/protocol-ping/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@libp2p/ping",
+  "version": "0.0.0",
```

Comment:
:eyes:

Reply:
Yes, this is a breaking change in this module so publishing will give us v1